mobify/platform-scaffold
# Fix base styles

Move styles that was in `_levels.scss` into proper place in base styles. 

 **JIRA**: https://mobify.atlassian.net/browse/AMP-51
 **Linked PRs**: n/a

## Changes
- remove `_level.scss`
- move styles that was in `_level.scss` into proper place in base styles. 
- remove `!important` for `[hidden]`
- remove `!important` in button on PDP, and use add classes instead.
- wrap styles in template class,`t-onboarding`.

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Check all pages in MP to make sure that it looks same.
